dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -35,8 +35,9 @@ const contentWrapperStyle = css`
 `;
 
 const sectionWrapperStyle = (hide: boolean) => css`
-	display: ${hide ? 'none' : 'unset'};
+	display: ${hide ? 'none' : 'block'};
 	position: fixed;
+	position: -webkit-sticky;
```

Comment:
Thanks, that's good to know! The browser-specific css isn't needed here as we just need to add another fallback value  for the property rather than have an alternative implementation. I've added a disable comment with a note about why its there.